sdcm/nemesis.py

```diff
@@ -73,6 +73,8 @@ class Nemesis():  # pylint: disable=too-many-instance-attributes,too-many-public
     disruptive = False
     run_with_gemini = True
     networking = False
+    HOUR_IN_SEC = 60 * 60
+    SIX_HOURS_IN_SEC = HOUR_IN_SEC * 6
```

Comment:
better to define 
HOUR = 3600
MINUTE = 60
and then use it all over the code
10 * HOUR 
5 * MINUTE
or use an external package.

Reply:
fixed